<launch>
  <arg name="pub_freq" default="10.0"/>

    <node pkg="multi_target_tracking" name="pursuer_evader_data_node" type="pursuer_evader_data.py" output="screen">
        <param name="num_pursuer_drones" type="int" value="2"/>
        <param name="num_evader_drones" type="int" value="2"/>

        <!-- <param name="num_pursuer_drones" type="int" value="8"/>
        <param name="num_evader_drones" type="int" value="12"/> -->

        <!-- <param name="num_pursuer_drones" type="int" value="10"/>
        <param name="num_evader_drones" type="int" value="15"/> -->

        <!-- <param name="detection_range" type="int" value="60"/> -->
        <param name="detection_range" type="int" value="45"/>
        <param name="communication_range" type="int" value="150"/>

        <!-- <param name="horizontal_FOV" type="double" value="2.1"/>
        <param name="vertical_FOV" type="double" value="1.8"/> -->

        <param name="horizontal_FOV" type="double" value="1.571"/>
        <param name="vertical_FOV" type="double" value="1.274"/>

        <param name="pub_freq" type="double" value="$(arg pub_freq)"/>  
    </node>
</launch>